.travis.yml

```diff
@@ -1,34 +1,48 @@
 language: python
+sudo: false
+dist: bionic
+
 cache: pip
 
+python:
+  - "3.6"
+  - "3.7"
+
 addons:
   apt:
     packages:
       - sqlite3
+      - fping
       - gdal-bin
       - libproj-dev
       - libgeos-dev
       - libspatialite-dev
+      - spatialite-bin
+      - libsqlite3-mod-spatialite
 
-python:
-  - "3.6"
+services:
+  - docker
+  - redis-server
 
 branches:
   only:
     - master
 
 before_install:
+  - docker run -d --name influxdb -p 8086:8086 influxdb
+  - docker exec -it influxdb influx -execute 'CREATE DATABASE openwisp2'
   - pip install -U pip wheel setuptools
-  - pip install --no-cache-dir -U -r requirements-test.txt
-  - ./runflake8
-  - ./runisort
+  - pip install --no-cache-dir -U -r requirements.txt -r requirements-test.txt
```

Comment:
install only `requirements-test.txt`, don't worry if it fails, I will help you debug it.